pydatastructs/linear_data_structures/arrays.py

```diff
@@ -148,6 +148,7 @@ def fill(self, elem):
 
     def __len__(self):
         return self._size
+    
```

Comment:
```suggestion
```

Reply:
Sorry, I didnt get this

Reply:
It's a suggestion to remove that extra line. It's not needed.